package.json

```diff
@@ -43,8 +43,8 @@
     "lint-quiet:es": "eslint 'packages/**/*.ts{,x}' --cache --quiet",
     "lint:ts": "tsc",
     "lint-staged": "yarn exec lint-staged",
-    "lint-changed": "eslint $(git diff --name-only --diff-filter=ACMRTUXB main | grep  -E \"(.js$|.ts$|.tsx$)\") --quiet",
-    "lint-changed-fix": "eslint $(git diff --name-only --diff-filter=ACMRTUXB main | grep  -E \"(.js$|.ts$|.tsx$)\") --quiet --fix",
+    "lint-changed": "eslint $(git diff --name-only --diff-filter=ACMRTUXB origin/main | grep  -E \"(.js$|.ts$|.tsx$)\") --quiet",
+    "lint-changed-fix": "eslint $(git diff --name-only --diff-filter=ACMRTUXB origin/main | grep  -E \"(.js$|.ts$|.tsx$)\") --quiet --fix",
```

Comment:
unrelated but commit hooks failed and I had to `lint-changed-fix` and I never keep a local `main` branch around so here we are

Reply:
I think this is a better statement for the script anyway